circle.yml

```diff
@@ -39,3 +39,23 @@ deployment:
           -l "${CIRCLE_ARTIFACTS}"/acceptance_deploy 2>&1 >
           "${CIRCLE_ARTIFACTS}/acceptance_deploy.log"
       - build/push-aws.sh
+  osx:
+    branch: master
+    commands:
+      - |
+          if [ -n "${BUILD_OSX-}" ]; then
+            set -eux
+            # There must certainly be a better way to move ~/cockroach
+            # in its appropriate location in GOPATH (symlinking doesn't
+            # work due to path traversal issues on the build).
+            export repo="${GOPATH%%:*}/src/github.com/cockroachdb/cockroach"
+            cd "${repo}/.."
+            rm -r "${repo}"
+            mv ~/cockroach "${repo}"
```

Comment:
If the repo isn't in the right place, then we don't have our dependencies either. Is `xgo` running `go get` for us (in which case we're not getting our pinned dependency versions)

Oh - `make ldflags` will bootstrap. Sneaky.